.github/workflows/build-distribute.yml

```
@@ -0,0 +1,163 @@
+
+      - name: Decode keystore
+        run: |
+          echo "${{ secrets.KEYSTORE_FILE }}" | base64 --decode > ${{ github.workspace }}/keystore.jks
+          echo "Keystore path: ${{ github.workspace }}/keystore.jks"
+
+      - name: Create local.properties
+        run: echo "sdk.dir=$ANDROID_SDK_ROOT" > local.properties
+
+      - name: Test Build
+        run: |
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)